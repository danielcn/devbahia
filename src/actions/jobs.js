export function addJob(text) {
  return {
    type: 'ADD_JOB',
    text,
  }
}
