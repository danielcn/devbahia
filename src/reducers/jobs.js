export default function jobs(state = [], action) {
  // action = { type: ADD_JOB, text: 'Fazer caf√©' }

  switch(action.type) {
    case 'ADD_JOB':
      return [ ...state, {
        id: Math.random(),
        text: action.text
      }]
    default:
      return state;
  }
}
