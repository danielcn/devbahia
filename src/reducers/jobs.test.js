import jobs from './jobs';

describe('Event Snapshot', () => {
  test('jobs', () => {
    expect(jobs).toMatchSnapshot();
  });
});
