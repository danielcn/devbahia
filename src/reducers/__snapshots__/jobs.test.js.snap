// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event Snapshot jobs 1`] = `[Function]`;
